aggregators/1.0/modules/ROOT/pages/aggregators-troubleshooting.adoc

```diff
@@ -48,7 +48,8 @@ To begin troubleshooting Aggregator module, enable verbose logging to see the ex
 
 == Understand the Timeout Behavior in the Size-Based Aggregator
 
-For the *Size based aggregator*, the *Timeout* field indicates the maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to the value set in the *Max size* field, the aggregation is not considered complete. The wait time passed and the aggregation is reset. Take into account that you can use an *Aggregator listener* source to capture the aggregated values.
+For the *Size based aggregator*, the *Timeout* field indicates the maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to the value set in the *Max size* field, the aggregation is not considered complete. The wait time passed and the aggregation is reset. Take into account that you can use an *Aggregator listener* source to capture the elements that were aggregated before the timeout occurred.
```

Comment:
```suggestion
For *Size based aggregator*, the *Timeout* field indicates the maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to the value set in the *Max size* field, the aggregation is not considered complete and the aggregation is reset. Note that you can use an *Aggregator listener* source to capture the elements that were aggregated before the timeout occurred.
```
"take into account" might be hard to translate, because "take" is used idiomatically.